README.md

```diff
@@ -2,93 +2,132 @@
-5. In the <b><samp>Capture</samp></b> tab, start profiling by pressing <kbd>F5</kbd>.  To stop profiling, press <kbd>F5</kbd> again. You can either zoom time using <kbd>W</kbd> and <kbd>S</kbd> or <kbd>ctrl</kbd> + the scroll wheel.  You can also <kbd>ctrl</kbd>+<b><samp>right-click</samp></b> and drag to zoom to a specific time range. To scale the UI, press <kbd>ctrl</kbd> + <kbd>+</kbd>/<kbd>-</kbd>. Press <kbd>SPACE</kbd> to see the last 2 seconds of capture.
-6. You can select sections of the per-thread sampling event track to get a sampling report of your selection.
+1. Select a process in the list of currently running processes in the connection
+   setup dialog, and click **Start Session**.
+2. The list of loaded modules will appear at the top of the **Symbols** tab.
+3. Orbit tries to automatically retrieve debug information of the modules.
+   See [here](documentation/DOCUMENTATION.md#load-symbols) on how to load
+   symbols for modules Orbit failed to load. For successfully loaded module
+   symbols, the **Functions** tab will get populated.
+4. Select functions you wish to dynamically instrument in the **Functions** tab
+   by <kbd>Right-Click</kbd> and choosing **Hook**.
+5. Start profiling by pressing <kbd>F5</kbd>. To stop profiling, press
+   <kbd>F5</kbd> again. You can either zoom time using <kbd>W</kbd> and
+   <kbd>S</kbd> or <kbd>Ctrl</kbd> + the scroll wheel. You can also
+   <kbd>Ctrl</kbd>+<kbd>Right-Click</kbd> and drag to zoom to a specific time
+   range. To scale the UI, press <kbd>Ctrl</kbd> + <kbd>+</kbd>/<kbd>-</kbd>.
+   Press <kbd>SPACE</kbd> to see the last 2 seconds of capture.
+6. You can select sections of the per-thread sampling event track to get a
+   sampling report of your selection.
 
 ## Presets
 
-Once you have loaded the debug information for your modules and have chosen functions of interest, you can save your profiling preset so that you won't have to do this manually again.  To save a preset, go to <b><samp>File</samp></b> ⇒ <b><samp>Save Preset</samp></b>
+Once you have loaded the debug information for your modules and have chosen
+functions of interest to dynamically instrument, you can save your profiling
+preset so that you won't have to do this manually again. To save a preset, go to
+**File** > **Save Preset**
 
 ### Feedback
 
-Questions and comments are more than welcome, please open an [issue](https://github.com/google/orbit/issues).
+Questions and comments are more than welcome: please open an
+[issue](https://github.com/google/orbit/issues/new).
 
 ## About
 
-Orbit was created by [Pierric Gimmig](https://www.linkedin.com/in/pgimmig/) but is now actively developed and maintained by a team of engineers at Google.
+Orbit was created by [Pierric Gimmig](https://www.linkedin.com/in/pgimmig/), but
+is now developed and maintained by a team of engineers at Google.
```

Comment:
Maybe at this point we should list the team in its entirety here?

Reply:
Done.